### DISCUSSION TITLE
Chatting platform

### DISCUSSION THREAD

**Discussion by zamazan4ik** [litmuschaos/litmus#3749] [Q&A] [answered]
Hi!

I am trying to find a place where I can chat about Litmus and some ways, how it could be improved. I guess Slack is such a place. But I cannot find the right link to the Slack. Since at least one link leads to kubernetes.slack.com which seems to be wrong space. Would be awesome if will share the right link to the space.

Thanks in advance!

**Comment by gdsoumya** [comment, marked as answer]
That is the correct slack workspace, try searching for `#litmus` channel in the k8s slack.

**Reply by zamazan4ik** [reply]
Thank you!

**Comment by Shreyash2607** [comment]
In local Development Guide , It is mentioned that to start authentication service I need to export environment Variable and then start the authentication service by GO. Can someone tell me how to export environment variables
